src/main/java/de/hpi/bpt/argos/core/Argos.java

```diff
@@ -63,6 +63,14 @@ static String getArgosBackendTestModePropertyKey() {
 	}
 
 	/**
+	 * This method returns the property key for the argosBackendEventTypeDirectory property.
+	 * @return - the property key for the argosBackendEventTypeDirectory property
+	 */
+	static String getArgosBackendEventTypeDirectory() {
```

Comment:
Please change to `getArgosBackendEventTypeDirectoryPropertyKey` for consistency.